lib/android/app/src/main/java/com/reactnativenavigation/views/tooltips/TooltipsOverlay.kt

```diff
@@ -0,0 +1,84 @@
+package com.reactnativenavigation.views.tooltips
+
+import android.app.Activity
+import android.content.Context
+import android.graphics.Color
+import android.view.Gravity
+import android.view.MotionEvent
+import android.view.View
+import android.widget.FrameLayout
+import android.widget.TextView
+import com.reactnativenavigation.utils.ColorUtils
+import com.reactnativenavigation.views.ViewTooltip
+
+class TooltipsOverlay(context: Context, id: String, debug:Boolean = false) : FrameLayout(context) {
+
+    init {
+        if(debug){
+            addView(TextView(context).apply {
+                text = id
+                textSize = 18f
+            }, LayoutParams(LayoutParams.WRAP_CONTENT, LayoutParams.WRAP_CONTENT).apply {
+                gravity = Gravity.CENTER
+            })
+            setBackgroundColor(ColorUtils.randomColor(0.25f))
+        }
```

Comment:
can be removed?

Reply:
yes but, it is needed for debug purposes